tests/functional/z_cluster/test_scale_mon_pods.py

```diff
@@ -0,0 +1,324 @@
+            try:
+                assert self.storagecluster_obj.patch(
+                    params=params_neg,
+                    format_type="merge",
+                ), log.error("Mon count should not be updated value other than 3 and 5")
+            except CommandFailed:
+                log.info("Mon count cannot not be updated value other than 3 and 5")
+
+            self.storagecluster_obj.patch(
+                params=params,
+                format_type="merge",
+            )
+
+            log.info("Verifying that all five mon pods are in running state")
+            assert verify_mon_pod_running(
+                self.mon_count
+            ), "All five mon pods are not up and running state"
+
+            self.ceph_cluster.cluster_health_check(timeout=60)
+
+            measure_end_time = time.time()
+
+            assert len(list_mons) != self.mon_count, pytest.skip(
+                "INVALID: Mon count is already set to five."
+            )
+        else:
+            # If CephMonLowCount warning is not found post updating mon count, fail the test
+            pytest.fail(
+                "Failed to get CephMonLowCount warning when mon count is updated to five"
+            )
+
+        log.info(
+            f"Verify that CephMonLowNumber alert got cleared post updating monCount to {self.mon_count}"
+        )
+        api.check_alert_cleared(
+            label=target_label, measure_end_time=measure_end_time, time_min=300
+        )
+
+    @ignore_leftovers
+    def test_mon_restart_post_five_mon_update(
+        self, nodes, pvc_factory, pod_factory, bucket_factory, rgw_bucket_factory
+    ):
+        """
+        Test nodes restart (from the platform layer, i.e, EC2 instances, VMWare VMs) post
+        updating the monCount to five
+
+        """
+
+        list_mons = self.ceph_cluster.get_mons_from_cluster()
+        if len(list_mons) < 5:
+            params = '{"spec":{"managedResources":{"cephCluster":{"monCount": 5}}}}'
+            self.storagecluster_obj.patch(
+                params=params,
+                format_type="merge",
+            )
+            log.info("Verifying that all five mon pods are in running state")
+            assert verify_mon_pod_running(
+                self.mon_count
+            ), "All five mon pods are not up and running state"
+
+            self.ceph_cluster.cluster_health_check(timeout=60)
+
+        ocp_nodes = get_node_objs()
+        if is_vsphere_ipi_cluster():
+            # When using vSphere IPI, we restart the nodes without stopping them.
+            # See issue https://github.com/red-hat-storage/ocs-ci/issues/7760.
+            nodes.restart_nodes(nodes=ocp_nodes, force="True", wait=False)
+            node_names = [n.name for n in ocp_nodes]
+            wait_for_nodes_status(node_names, constants.STATUS_READY, timeout=420)
+        else:
+            nodes.restart_nodes_by_stop_and_start(nodes=ocp_nodes, force="True")
```

Comment:
Please change to `nodes.restart_nodes_by_stop_and_start(nodes=ocp_nodes, force=True)`